src/core/diags/factory_diags.cpp

```diff
@@ -429,6 +430,18 @@ Error Diags::ProcessSend(uint8_t aArgsLength, char *aArgs[])
 
     VerifyOrExit(aArgsLength >= 1, error = kErrorInvalidArgs);
 
+    if (StringMatch(aArgs[0], "async"))
```

Comment:
```suggestion
    if (StringMatch(aArgs[0], "-a"))
```

Reply:
I just follow the general usage of `async` in cli commands. You can search `async` in [src/cli/README.md](https://github.com/openthread/openthread/blob/main/src/cli/README.md) to see the general usage.